halpybot/packages/models/case.py

```diff
@@ -0,0 +1,71 @@
+"""
+case.py - The Case Object
+
+Copyright (c) The Hull Seals,
+All rights reserved.
+
+Licensed under the GNU General Public License
+See license.md
+"""
+from typing import Optional
+from enum import Enum
+from attrs import define, field
+from attr import dataclass
+import pendulum
+
+
+class Platform(Enum):
+    """Storing Platform References"""
+
+    ODYSSEY = 1
+    XBOX = 2
+    PLAYSTATION = 3
+    LEGACY_HORIZONS = 4
+    LIVE_HORIZONS = 5
+
+
+class Status(Enum):
+    """Saving Case Status"""
+
+    ACTIVE = 0
+    CLOSED = 1
+    DELAYED = 2
+
+
+@dataclass
```

Comment:
Inconsistent usage of `@dataclass` and `@define` within this module.
prefer `define` as its the newer API.

Reply:
Was unaware they were basically the same. Updated.